src/panels/lovelace/editor/card-editor/hui-edit-card.ts

```diff
@@ -198,7 +198,7 @@ export class HuiEditCard extends LitElement {
     if (!this._uiEditor) {
       setTimeout(() => {
         this.yamlEditor.codemirror.refresh();
-      }, 1);
+      }, 50);
```

Comment:
What is it that we're waiting for, and can we wait for that specifically? (like maybe `this.updateComplete` ?). Because it really depends on the speed of your computer/phone etc if 50 is going to be enough.

Reply:
I think we are waiting for the dialog to be resized... the update of the element itself is already complete...